Tests/FluentAssertions.Specs/Primitives/DateTimeOffsetAssertionSpecs.cs

```diff
@@ -2476,7 +2476,7 @@ public void Should_throw_a_helpful_error_when_accidentally_using_equals_with_a_r
             DateTimeOffset someDateTimeOffset = new(2022, 9, 25, 13, 48, 42, 0, TimeSpan.Zero);
 
             // Act
-            Action action = () => someDateTimeOffset.Should().BeLessThan(0.Seconds()).Equals(someDateTimeOffset);
+            Func<bool> action = () => someDateTimeOffset.Should().BeLessThan(0.Seconds()).Equals(someDateTimeOffset);
```

Comment:
🔧 For tests, we need to disable this rule. We don't need the `bool` here.

Reply:
C# 10 introduced "natural types" for lambdas letting us writing `var` here. I'd be fine with that.

Reply:
Can we re-enable `ReturnValueOfPureMethodIsNotUsed`, now that we're using `var` here?
I'm sorry for the clunky communication on this.

Reply:
@dennisdoomen ^^

Reply:
Do you want this to be part of this PR? If so, @IT-VBFK can you take care of that?

Reply:
Revert this commit or using `var`?

Reply:
The changes that 1882b0dd72432cf939d828e6d95bbb4ed12b7979 did to `qodana.yaml`